addons/crm/crm_lead.py

```diff
@@ -730,28 +730,35 @@ def convert_opportunity(self, cr, uid, ids, partner_id, user_ids=False, section_
 
         return True
 
+    @api.one
```

Comment:
@api.one is also deprecated in v9, so it should be removed from code as soon as you can.